### PR TITLE
Code suggestions for Mouna

### PR DESCRIPTION
**Pull request by cmadjar**
I'll comment directly on the files changes to explain why I implemented those little changes

### CODE REVIEW

File: tools/MakeNiiFilesBIDSCompliant.pl [hunk 1]
```diff
@@ -358,6 +358,11 @@ sub makeNIIAndHeader {
         my $sth = $dbh->prepare($query);
         $sth->execute($fileAcqProtocolID);
         my $rowhr               = $sth->fetchrow_hashref();
+        unless ($rowhr) {
```
Comment (cmadjar) [inline tools/MakeNiiFilesBIDSCompliant.pl:361]: I added this as I don't want to convert DTISiemens maps into BIDS so this query returns nothing and then it creates a lot of errors down. So if the query did not return anything, I go to the next file

File: tools/MakeNiiFilesBIDSCompliant.pl [hunk 2]
```diff
@@ -492,13 +497,11 @@ sub makeNIIAndHeader {
...
-                    ($headerNameMINC eq 'acquisition:inversion_time') ||
-                    ($headerNameMINC eq 'dicom_0x0018:el_0x1314')) {
-                        $headerVal *= 1;
-                }
-
+                my @convertToFloat = [
+                    'acquisition:repetition_time', 'acquisition:echo_time',
+                    'acquisition:inversion_time',  'dicom_0x0018:el_0x1314'
+                ];
+                $headerVal *= 1 if ($headerVal && $headerNameMINC ~~ @convertToFloat);
```
Comment (cmadjar) [inline tools/MakeNiiFilesBIDSCompliant.pl:504]: for this one, it happens sometimes that one of them returns undef that is after converted into a float returning an error but continuing with a new $headerVal=0. So I check first that there $headerVal is defined and check if the header is one of the header that needs conversion to float

File: tools/MakeNiiFilesBIDSCompliant.pl [hunk 3]
```diff
@@ -540,7 +543,7 @@ sub makeNIIAndHeader {
                 $extraHeader    = "SliceTiming";
                 $headerVal      =  &fetchMincHeader($mincFileName,$headerNameMINC);
                 $headerVal = [map {1 * $_} split(",", $headerVal)];
-                    print "    SliceTiming $headerVal was added \n" if $verbose;;
+                print "    SliceTiming $headerVal was added \n" if $verbose;;
```
Comment (cmadjar) [inline tools/MakeNiiFilesBIDSCompliant.pl:546]: my OCD talking again... ;)

File: tools/MakeNiiFilesBIDSCompliant.pl (hunk 2 above)
Comment (MounaSafiHarab) [reply tools/MakeNiiFilesBIDSCompliant.pl:504]: this was in my TODO list in the PR description, I guess I can cross it now:)

**Comment by MounaSafiHarab** [comment]
thanks @cmadjar !

**Pull request merged**